autotst/species_test.py

```diff
@@ -67,10 +68,11 @@ def test_get_angles(self):
         self.assertIsInstance(angles[0],Angle)
         self.assertEquals(len(angles),12)
     def test_get_torsions(self):
-        torsions = self.conformer.get_torsions()
-        self.assertIsInstance(torsions,list)
-        self.assertIsInstance(torsions[0],Torsion)
-        self.assertEquals(len(torsions),1)
+        for conformer in [self.conformer, self.conformer_torsion_test]:
+            torsions = conformer.get_torsions()
+            self.assertIsInstance(torsions,list)
+            self.assertIsInstance(torsions[0],Torsion)
+            self.assertEquals(len(torsions),1)
```

Comment:
Good test to check that we're finding the right number of torsions! Can you also add one to check the number of center atoms for the C-C#C-C case?